README.md

```diff
@@ -93,6 +93,23 @@ oneview_ethernet_network 'Eth1' do
 end
 ```
 
+### oneview_fabric
+
+Fabric resource for HPE OneView.
+
+Support only in API300 onwards with variant Synergy.
+
+Action defaults to `:none`, also performs updates on the reserved vlan range.
```

Comment:
Well, since the default is always `:create` or `:none` and the resource has no `:create` action, I put `:none`